### PR TITLE
0208 update settings to complete rollover

### PR DESCRIPTION
**Pull request by defong**
### Context
Rollover

### Changes proposed in this pull request
Rollovered

### Guidance to review
Best to view per commit.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

**Comment by defong** [comment]
[Update settings to complete rollover and set current cycle to 2021 (Publish)](https://trello.com/c/z7uMl6Dg/208-update-settings-to-complete-rollover-and-set-current-cycle-to-2021-publish)

File: config/routes.rb [hunk 1]
```diff
@@ -67,7 +67,7 @@
     end
 
     # TODO: Extract year constraint to future proof for future cycles
-    resources :recruitment_cycles, param: :year, constraints: { year: /2020|2021/ }, path: "", only: :show do
+    resources :recruitment_cycles, param: :year, constraints: { year: /2021|2022/ }, path: "", only: :show do
```
Comment (misaka) [inline config/routes.rb:70]: I wonder if `Settings` is loaded by this time ... it outta be. If so, we could use that in the constraint here.

File: spec/features/courses/preview_spec.rb [hunk 2]
```diff
@@ -160,9 +160,18 @@
 
     expect(preview_course_page).to_not have_salary_details
 
-    expect(preview_course_page.scholarship_amount).to have_content("a scholarship of £2,000")
-
-    expect(preview_course_page.bursary_amount).to have_content("a bursary of £4,000")
+    # START: finanical support placeholder
+    # NOTE: This is temporary.
```
Comment (misaka) [inline spec/features/courses/preview_spec.rb:164]: Is this more accurate? Might help any devs who aren't familiar with rollover better understand what's going on.

```suggestion
    # NOTE: This is temporary as the financial incentives for this cycle haven't been released yet.
```

File: spec/features/courses/preview_spec.rb [hunk 3]
```diff
@@ -160,9 +160,18 @@
...
+    expect(decorated_course.use_financial_support_placeholder?).to be_truthy
+
+    expect(preview_course_page.find(".govuk-inset-text"))
+      .to have_text("Bursaries, scholarships and financial support for 2021 to 2022 will be announced soon.")
+    expect(preview_course_page).to_not have_scholarship_amount
+    expect(preview_course_page).to_not have_bursary_amount
+    # NOTE: revert to this in due course.
```
Comment (misaka) [inline spec/features/courses/preview_spec.rb:171]:
```suggestion
    # NOTE: revert to this once we know the actual figures
```

File: config/routes.rb (hunk 1 above)
Comment (defong) [reply config/routes.rb:70]: Carded as https://trello.com/c/jwC8kKnb/373-todo-extract-year-constraint-to-future-proof-for-future-cycles

We can wonder then... it should be fine... but its out of scope for this PR as it is big enough.

**Review by dankmitchell — APPROVED** [review]

**Review by gpeng — APPROVED** [review]

**Comment by defong** [comment]
[When cycle ends, move the "Next cycle" create PE forms to "current cycle"](https://trello.com/c/kLNgJpjs/211-when-cycle-ends-move-the-next-cycle-create-pe-forms-to-current-cycle)

**Pull request merged**